.maintain/playbooks/run-integration-tests.yml

```diff
@@ -0,0 +1,202 @@
+    apt_repository:
+      repo: "deb https://dl.yarnpkg.com/debian/ stable main"
+      filename: yarn
+    become: yes
+
+  - name: Download Node v16 installer
+    get_url:
+      url: https://deb.nodesource.com/setup_16.x
+      dest: "/tmp/setup_node.sh"
+      mode: '0440'
+    become: yes
+
+  - name: Add Node repository
+    ansible.builtin.shell: |
+      cat /tmp/setup_node.sh | bash
+    become: yes
+
+  - name: Update repositories cache and install nodejs and yarn
+    apt:
+      pkg:
+      - yarn
+      - nodejs
+      update_cache: yes
+      install_recommends: no
+    become: yes
+
+  - name: Clone composable repo
+    ansible.builtin.git:
+      repo: "https://{{ github_user | urlencode }}:{{ github_password | urlencode }}@github.com/ComposableFi/composable.git"
+      dest: "{{ work_dir }}/composable"
+      version: main
+    become: yes
+    become_user: "{{ user }}"
+
+  - name: Clone polkadot repo
+    ansible.builtin.git:
+      repo: https://github.com/paritytech/polkadot.git
+      dest: "{{ work_dir }}/polkadot"
+      version: v0.9.13
+    become: yes
+    become_user: "{{ user }}"
+
+  - name: Clone Basilisk-node repo
+    ansible.builtin.git:
+      repo: https://github.com/galacticcouncil/Basilisk-node.git
+      dest: "{{ work_dir }}/Basilisk-node"
+      version: v5.0.3
+    become: yes
+    become_user: "{{ user }}"
+
+  - name: Install dependencies of polkadot-launch
+    ansible.builtin.shell: yarn
+    args:
+      chdir: "{{ work_dir }}/composable/scripts/polkadot-launch"
+      executable: /bin/bash
+    become: yes
+    become_user: "{{ user }}"
+
+  - name: Install dependencies of integration tests and build integraion tests
+    ansible.builtin.shell: |
+      yarn
+      yarn build
+    args:
+      chdir: "{{ work_dir }}/composable/scripts/polkadot-launch/initialization"
+      executable: /bin/bash
+    become: yes
+    become_user: "{{ user }}"
+
+  - name: Create systemd service
+    template:
+      src: systemd/composable-sandbox.j2
+      dest: /etc/systemd/system/composable-sandbox.service
+      mode: "0600"
+    become: yes
+
+  - name: Stop composable-sandbox service, if running
+    ansible.builtin.systemd:
+      name: composable-sandbox
+      state: stopped
+      daemon_reload: yes
+      enabled: true
+    become: yes
+
+  - name: Compile composable
+    ansible.builtin.shell: |
+      source $HOME/.cargo/env
+      cargo build --release
+    args:
+      chdir: "{{ work_dir }}/composable"
+      executable: /bin/bash
+    become: yes
+    become_user: "{{ user }}"
+
+  - name: Compile polkadot
+    ansible.builtin.shell: |
+      source $HOME/.cargo/env
+      cargo build --release
+    args:
+      chdir: "{{ work_dir }}/polkadot"
+      executable: /bin/bash
+    become: yes
+    become_user: "{{ user }}"
+
+  - name: Compile Basilisk-node
+    ansible.builtin.shell: |
+      source $HOME/.cargo/env
+      cargo build --release
+    args:
+      chdir: "{{ work_dir }}/Basilisk-node"
```

Comment:
they have specific deps on rust, would not that require more rust intall https://github.com/galacticcouncil/Basilisk-node/blob/master/rust-toolchain ? or it is automatic?

Reply:
yes, nightly-2021-11-07 was installed automatically 
```
$ rustup toolchain list
stable-x86_64-unknown-linux-gnu (default)
nightly-2021-08-22-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu
```